### PR TITLE
mantine to tailwind, responsive and some small changes

### PR DESCRIPTION
**Pull request by Temucle**

### CODE REVIEW

File: src/bazo/about_us.tsx [hunk 1]
```diff
@@ -89,312 +76,129 @@ export const About = () => {
...
-          "Хамгийн агуу сэтгэл бол бусдын төлөө сэтгэл"
-        </h2>{' '}
-      </div>
+      </section>
 
       {/* second section */}
-      <section className="w-full bg-[url('/images/media/firstBG.png')] bg-cover pb-16 pt-16">
-        <div>
-          <Text fw={700} size="34px" p={10} ta="center">
+      <section className="w-full bg-[url('/images/media/firstBG.png')] bg-cover py-16">
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:91]: background linear gradient bolgoyoo hairan storage :)

File: src/bazo/about_us.tsx [hunk 2]
```diff
@@ -478,352 +279,152 @@ export const About = () => {
...
-            </Text>
-            <Text>
-              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam,
-              officiis! Fugit minus ea, perferendis eum consectetur quae vitae.
-              Aliquid, quam reprehenderit? Maiores sed pariatur aliquid commodi
-              atque sunt officiis natus?
-            </Text>
+            <img
+              src="/images/about/AboutFam1.png"
+              alt="all in one"
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:297]: all in one gesen zuragnud ni ajilkuen :)

File: src/bazo/about_us.tsx [hunk 3]
```diff
@@ -478,352 +279,152 @@ export const About = () => {
...
+              className=""
+            />
+          </div>
+          <div className="flex w-[511px] flex-col md:my-4 ">
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:301]: width < flex-1 ashiglayaa

File: src/bazo/about_us.tsx [hunk 4]
```diff
@@ -478,352 +279,152 @@ export const About = () => {
...
-                width: 307,
-                height: 358,
-                marginLeft: 'auto',
-                marginRight: 'auto',
-              }}
-              className="rounded-md bg-white p-4"
-            >
+      </div>
+
+      <div className="bg-[url('/images/media/secondBG.png')] py-16 md:py-28">
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:369]: linear gradient

Comment (Enhmunh-E) [reply src/bazo/about_us.tsx:369]: Heterhii shagagdchad bn joohon zai uldeehgu bol
Screenshot 2024-05-11 at 10 55 23

File: src/bazo/about_us.tsx [hunk 5]
```diff
@@ -89,312 +76,129 @@ export const About = () => {
...
-              </ul>
-              <Link href="#">
-                <Button className="relative bottom-5 h-[39px] w-[240px] rounded-xl bg-primarySecond text-lg text-brand450  hover:bg-primarySecond">
-                  Гишүүн болох
-                </Button>
-              </Link>
+        <div>
+          <div className="mx-auto">
+            <div className="my-9 flex justify-center">
+              <button
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:107]: ene hamtragch demjigch gedgiin style iig joohon improve hiiya

File: src/bazo/about_us.tsx [hunk 6]
```diff
@@ -89,312 +76,129 @@ export const About = () => {
...
+                <span className="text-white relative bottom-14 left-7 text-start text-xl font-extrabold">
                   Standart Plan
                 </span>
                 <ul className="text-md relative bottom-14 w-[322px] pl-7 text-start text-gray-200">
                   Та санд төсөл хөтөлбөр болон арга хэмжээ оруулах,сурталчилгаа
                   байршуулах болон өөрийн нэр дээрх хуудсыг хөгжүүлээрэй
                 </ul>
                 <Link href="#">
-                  <Button className="bg-primary border-1.5 border-primary text-brand450 relative bottom-5 right-16 h-[48px] w-[182px] rounded-xl text-lg font-bold  hover:bg-transparent">
+                  <button className="bg-primary border-1.5 border-primary text-white relative bottom-5 right-16 h-[48px] w-[182px] rounded-xl text-lg font-bold  hover:bg-transparent">
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:196]: buttnguudad width height uguhgu shu, paddingaar l bolgoh heregtei

File: src/bazo/about_us.tsx [hunk 7]
```diff
@@ -478,352 +279,152 @@ export const About = () => {
...
+      </section>
 
-      {/* used to mantin ui */}
-      {/* <section>
-        <div style={{ display: 'flex' }}>
-          <AspectRatio ratio={1} style={{ flex: `0 0 ${rem(100)}` }}>
-            <Image src="/images/about/AboutFam1.png" alt="about" />
-          </AspectRatio>
+      <div className="flex w-full flex-col py-10 md:py-28 md:gap-16">
+        <div className="flex justify-start gap-8 md:gap-16 bg-black/5 container max-md:py-8">
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:293]: design deer background uguugu bsishd, debuglaj bga bol ustgarai

File: src/bazo/about_us.tsx [hunk 8]
```diff
@@ -1,79 +1,66 @@
...
+          <div className="text-lg max-md:order-2 md:w-[55%]">
+            <p className="text-primary text-center font-bold mb-2">
+              "Хамгийн агуу сэтгэл бол бусдын төлөө сэтгэл"
+            </p>
+            <p className="font-medium">
+              {' '}
+              Бид хэдий чинээ эрт хамтран ажиллана, төдий чинээ эрт зөв тусыг
+              хэрэгтэй хүнд нь <br /> хүргэх, нийгмийн хэт туйлшрал, бэлэнчлэх
+              сэтгэлгээ, ядуурал, өвчлөлөөс хүн амыг <br /> хамгаалах боломжтой.{' '}
+              <br /> Иймд бид НҮТББ, хувь хүн, энэ төрлийн үйл ажиллагаа
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:21]: br tagnuuda ustgaad uguulber ni shine paragraph ehelj baih uyd l br ashiglarai

File: src/bazo/about_us.tsx [hunk 9]
```diff
@@ -1,79 +1,66 @@
...
+              дурынхны зохион байгуулдаг хүмүүнлэг болон сайн <br /> үйлсийн
+              аян, төсөл, хөтөлбөрийг нэгтгэсэн Mонголын анхны ALL IN ONE олон
+              талт дэмжих <br /> системийг хайр түгээгч та бүхэндээ зориулан
+              хөгжүүлж байна. <br /> Уг сангийн одоогийн хувилбар нь эцсийн
+              хувилбар биш бөгөөд цаашид та бүхний саналын <br /> дагуу 4 талт
+              оролцогчдод давуу байдлаар тасралтгүй хөгжүүлсээр байх болно.
+            </p>
+          </div>
+          <div className="md:w-[45%] flex justify-center max-md:order-1 items-center">
+            <img
```
Comment (Enhmunh-E) [inline src/bazo/about_us.tsx:32]: ingej hiisen zuragnuud ni deer dooro bsih align hiitsn bshd
Screenshot 2024-05-11 at 10 59 42

**Review by Enhmunh-E — CHANGES_REQUESTED** [review]

File: src/bazo/about_us.tsx (hunk 3 above)
Comment (Temucle) [reply src/bazo/about_us.tsx]: resolved

**Pull request merged**